_data/fellows.yml

```diff
@@ -1,15 +1,31 @@
 - name: Will
   img: logo.jpg
   location: London, UK
+  linkedin: https://www.linkedin.com/in/prakhar-rathi/
+  twitter: https://twitter.com/PrakharRathi14
+  github: https://github.com/prakharrathi25
+  resume: 
 
 - name: Amanda
   img: amanda.jpg
```

Comment:
I don't think this is needed. Adding a fix for this as well.

Reply:
Could you elaborate on why this isn't needed? And also please add what the high level fix is.

Reply:
@arteevraina if you could add a little more detail in your reviews it would help us understand your thought process and have a more meaningful discussion.